de.fu_berlin.inf.dpp.whiteboard/frontend/package.json

```diff
@@ -2,41 +2,43 @@
   "name": "whiteboard",
   "version": "0.0.1",
   "description": "Whiteboard for Saros built with HTML",
+  "author": "Abdullah Barhoum",
```

Comment:
We generally don't use author tags in the Saros sourcecode.

Reply:
I had it there even before saros, I will remove it in the next commit.